install/operator/pkg/syndesis/template/install.go

```diff
@@ -160,6 +159,16 @@ func SetupRenderContext(renderContext *generator.Context, syndesis *v1alpha1.Syn
 		return fmt.Errorf("required config var not set: %s", configuration.EnvSarNamespace)
 	}
 
+	//
+	// Apply DevImages flag value to Debug
+	//
+	// If using dev images then doing development so
+	// useful to have JAVA_DEBUG set to allow for image debugging
+	//
+	if syndesis.Spec.DevImages != nil {
```

Comment:
Perhaps this would be a good opportunity to rename `DevImages` to `DevSupport` (like we already have `TestSupport`?